client/components/tinymce/plugins/contact-form/dialog/field-edit-button.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import Popover from 'components/popover';
+
+export default React.createClass( {
+	displayName: 'ContactFormDialogFieldEditButton',
```

Comment:
We use a babel transform that picks this up from the class or function name. We generally don't need to set this unless it needs to be different from the class or function name.